adapters/33across/33across.go

```diff
@@ -104,36 +110,35 @@ func preprocess(request *openrtb.BidRequest) error {
 	impExt.Ttx.Prod = ttxExt.ProductId
 
 	// Add zoneid if it's defined
+	impExt.Ttx.Zoneid = ttxExt.SiteId
```

Comment:
Please note that we will no longer be setting GUID under `site.id`, will use zoneID. This is to ensure that we can pass requests from `app` to TTX as well. TTX can decide independently how to handle this scenario,